package.json

```diff
@@ -1,15 +1,13 @@
 {
     "name": "dkehrig.show-whitespace",
     "title": "Show Whitespace",
-    "description": "Visualizes whitespace similar to how Sublime Text does it",
+    "description": "Visualize whitespace (both spaces and tabs) in the same style as Sublime Text. Now with configurable whitespace colors!",
```

Comment:
Ambiguous terms are ambiguous. :P You're saying to remove the notable changes highlight?